### PR TITLE
expand partial_eval to take boolean

### PR DESCRIPTION
**Pull request by yamaguchi1024**

### CODE REVIEW

File: SYS_ATL/LoopIR_scheduling.py
Comment (alexreinking) [inline SYS_ATL/LoopIR_scheduling.py:559]: Bool is a subtype of int in Python. This should just be `not isinstance(v, int)` and a comment should be left about intentionally excluding bool, too.

**Review by alexreinking — CHANGES_REQUESTED** [review]
One bit of code cleanup needed, but otherwise LGTM

File: SYS_ATL/LoopIR_scheduling.py
Comment (yamaguchi1024) [reply SYS_ATL/LoopIR_scheduling.py:559]: I see, that makes sense

**Review by alexreinking — APPROVED** [review]

**Comment by alexreinking** [comment]
LGTM - go ahead and merge when the checks pass

**Comment by yamaguchi1024** [comment]
> LGTM - go ahead and merge when the checks pass

Thanks!

**Pull request merged**
